shell/package.json

```diff
@@ -49,6 +49,7 @@
     "@nuxtjs/style-resources": "1.2.1",
     "@nuxtjs/webpack-profile": "0.1.0",
     "@popperjs/core": "2.4.4",
+    "@rancher/components": "0.1.0-alpha.7",
```

Comment:
I comfortable with graduating from alpha. I had originally tagged it as such to serve as a test bed for Rancher Desktop integration. We haven't experienced any issues with the library since its integration.